### PR TITLE
Nutmeg Upgrade - Code Merge 4/6

### PR DESCRIPTION
**Pull request by shadinaif**
## Change description

Nutmeg Upgrade - Code Merge 4/6

## Type of change
- [ ] Bug fix (fixes an issue)
- [x] New feature (adds functionality)

## Related issues

Related to: https://appsembler.atlassian.net/browse/RED-3371

### Development

- [ ] Lint rules pass locally
- [ ] Application changes have been tested thoroughly
- [ ] Automated tests covering modified code pass

### Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### Code review 

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [ ] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

File: lms/templates/dashboard/_dashboard_certificate_information.html [hunk 1]
```diff
@@ -47,26 +47,11 @@
       </p>
     </div>
   % else:
-<<<<<<< HEAD
```
Comment (OmarIthawi) [inline lms/templates/dashboard/_dashboard_certificate_information.html]: Checking why this conflict appeared in the first place. I'm guessing it's something we've fixed previously:

```diff
$ git diff main..open-release/juniper.master --  lms/templates/dashboard/_dashboard_certificate_information.html
diff --git a/lms/templates/dashboard/_dashboard_certificate_information.html b/lms/templates/dashboard/_dashboard_certificate_information.html
index 52985348ca..62f4b81bb8 100644
--- a/lms/templates/dashboard/_dashboard_certificate_information.html
+++ b/lms/templates/dashboard/_dashboard_certificate_information.html
@@ -48,7 +48,7 @@ else:
       <p class="message-copy">
         % if should_display_grade(course_overview):
             ${_("Your final grade:")}
-            <span class="grade-value">${"{0:.0f}%".format(float(cert_status.get('grade', 0))*100)}</span>.
+            <span class="grade-value">${"{0:.0f}%".format(float(cert_status['grade'])*100)}</span>.
         % elif course_overview.certificate_available_date:
           <%
             cert_available_date = course_overview.certificate_available_date.strftime('%Y-%m-%d')
```

Apparently we no longer needs this fix.

File: .github/workflows/migrations-check.yml [hunk 2]
```diff
@@ -3,6 +3,8 @@ name: CI
 on:
   workflow_dispatch:
   pull_request:
+    branches:
+      - master
```
Comment (OmarIthawi) [inline .github/workflows/migrations-check.yml:7]: Add a note: `- master  # Tahoe: this line effectively disables it on Tahoe`

**Review by OmarIthawi — APPROVED** [review]
Looks good to me as long as tests passes. Thanks a lot Shadi!

File: lms/templates/dashboard/_dashboard_certificate_information.html (hunk 1 above)
Comment (shadinaif) [reply lms/templates/dashboard/_dashboard_certificate_information.html]: Thank you Omar!

**Pull request merged**